docs/replication-manager/single-replica-manager.md

```diff
@@ -0,0 +1,33 @@
+# Single replica manager for Percona XtraDB Cluster
+
+The single replica manager is a simplified version of the [Replication manager for Percona XtraDB Cluster](replication-manager-for-pxc.md) and intended to manage a single replica. 
+
+The single replica manager requires using a global transaction identifier (GTID) (Oracle implementation).
```

Comment:
What is GTID? It's worth to add it to glossary IMHO

Reply:
Added